docs/07-Quick Starters/02-PHP/_category_.json

```diff
@@ -0,0 +1,3 @@
+{
+  "label": "PHP"
```

Comment:
- folder name should use lowercase
- add `collapse: false` (https://docusaurus.io/docs/next/sidebar/autogenerated#category-item-metadata)

Reply:
https://stackoverflow.com/questions/10523849/how-do-you-change-the-capitalization-of-filenames-in-git